plugins/@grouparoo/pardot/src/lib/export/destinationMappingOptions.ts

```diff
@@ -0,0 +1,118 @@
+import {
+  DestinationMappingOptionsMethod,
+  DestinationMappingOptionsResponseProperty,
+  DestinationMappingOptionsResponseTypes,
+} from "@grouparoo/core";
+
+import PardotClient from "../client";
+import { connect } from "./../connect";
+
+// Prospect field spec https://developer.pardot.com/kb/object-field-references/#prospect
+const requiredProspectFields: DestinationMappingOptionsResponseProperty[] = [
+  { key: "email", type: "email" },
+];
+
+const defaultProspectFields: DestinationMappingOptionsResponseProperty[] = [
+  { key: "campaign_id", type: "integer" }, // TODO check for validation
+  { key: "salutation", type: "string" },
+  { key: "first_name", type: "string", important: true },
+  { key: "last_name", type: "string", important: true },
+  { key: "password", type: "string" },
+  { key: "company", type: "string" },
+  { key: "prospect_account_id", type: "integer" }, // TODO: check for validation
+  { key: "website", type: "string" },
+  { key: "job_title", type: "string" },
+  { key: "department", type: "string" },
+  { key: "country", type: "string" },
+  { key: "address_one", type: "string" },
+  { key: "address_two", type: "string" },
+  { key: "city", type: "string" },
+  { key: "state", type: "string" },
+  { key: "territory", type: "string" },
+  { key: "zip", type: "string" },
+  { key: "phone", type: "phoneNumber" },
+  { key: "fax", type: "phoneNumber" },
+  { key: "source", type: "string" },
+  { key: "annual_revenue", type: "string" },
+  { key: "employees", type: "string" },
+  { key: "industry", type: "string" },
+  { key: "years_in_business", type: "string" },
```

Comment:
sure about this one. looks like integer?

Reply:
Yeah there are some weird types there that don't make sense to me. They're documented here https://developer.pardot.com/kb/object-field-references/#prospect

![image](https://user-images.githubusercontent.com/4368928/109698610-f6f38580-7b65-11eb-8e5a-03c41b6c4bfd.png)